database/data.json

```diff
@@ -1,10 +1 @@
-{
-  "posts": [
-    {
-      "id": 1,
-      "text": "Some text post 1",
-      "title": "Some title post 1"
-    }
-  ],
-  "comments": [{ "id": 1, "title": "Comment1", "text": "Comment1", "createdDate": 0, "postId": 1 }]
-}
+{"posts":[{"id":1,"text":"WAZZAp","title":"Some title post 1"},{"text":"some text2","title":"some title2","id":"03463cf5-372a-43c2-a58b-9b40dd1185cb"},{"text":"some text2444","title":"some title212124","id":"ee5c5d9a-1fcc-435f-be29-5b4aeff3e213"}],"comments":[{"id":1,"title":"Comment1","text":"Comment1","createdDate":0,"postId":1}]}
```

Comment:
try third parameter in JSON.stringify (or whatever else) to beautify it for yourself